WORKSPACE

```diff
@@ -18,6 +18,21 @@ http_archive(
     urls = ["https://github.com/protocolbuffers/protobuf/archive/v23.4.tar.gz"],
 )
 
+http_archive(
+    name = "googleapis",
+    sha256 = "9d1a930e767c93c825398b8f8692eca3fe353b9aaadedfbcf1fca2282c85df88",
+    strip_prefix = "googleapis-64926d52febbf298cb82a8f472ade4a3969ba922",
+    urls = [
+        "https://github.com/googleapis/googleapis/archive/64926d52febbf298cb82a8f472ade4a3969ba922.zip",
+    ],
+)
+
+load("@googleapis//:repository_rules.bzl", "switched_rules_by_language")
+
+switched_rules_by_language(
+    name = "com_google_googleapis_imports",
```

Comment:
Don't you need to set `go = true`?

https://github.com/googleapis/googleapis/blob/master/repository_rules.bzl#L87

Reply:
It seems we don't need this, because we use `@googleapis` only in `proto_library`. Go deps are resolved to `@org_golang_google_genproto_googleapis_api`. 

I suppose that I could change:

```
# gazelle:resolve proto go google/api/annotations.proto  @org_golang_google_genproto_googleapis_api//annotations
```

to

```
# gazelle:resolve proto go google/api/annotations.proto  @googleapis//google/api:annotations_go_proto
```

then `go = True` would be necessary. But I am not sure if it gives us any benefits. `@org_golang_google_genproto_googleapis_api//annotations` was recommended in `go_rules` and `gazelle`.